### DISCUSSION TITLE
How to debug

### DISCUSSION THREAD

**Discussion by asaf** [traceloop/openllmetry#1119] [Q&A] [answered]
Hey @nirga this is a reference from comment:  https://github.com/traceloop/openllmetry/issues/1096#issuecomment-2123582368

I started a local docker instance of `jaegertracing/all-in-one:1.57` to check out _openllmetry_ locally and I get `Failed to export batch code: 404, reason: 404 page not found` with the configuration suggested above.

I init Traceloop as you mentioned above:

```
Traceloop.init(
    app_name="Langchain Chatbot Application",
    api_endpoint="http://localhost:4318",
    disable_batch=True,
)
```
I also tried _api_endpoint="http://localhost:4318/v1/traces", same result.
(posting directly to Jaeger works)

I wonder if there's a way to debug Traceloop to ensure that the correct HTTP calls are made, docs mentions debugging only for ts/js version (https://www.traceloop.com/docs/openllmetry/configuration#control-logging)

Best,
Asaf.

**Comment by nirga** [comment]
Hey @asaf! IIUC from the [docs](https://www.jaegertracing.io/docs/1.6/getting-started/), Jaeger doesn't support the OpenTelemetry protocol directly, so you need to spin off an [OpenTelemetry collector](https://opentelemetry.io/docs/collector/quick-start/) as well that will receive the traces and send them out to Jaeger.

**Reply by asaf** [reply]
Hey, _openllmetry_ is working with `jaegertracing/all-in-one:1.57` except a single call to `/api/metrics` that results 404.

**Comment by nirga** [comment, marked as answer]
Oh got it! @asaf  
It means that Jaeger doesn't support the OpenTelemetry Metrics endpoint. You can disable it by setting TRACELOOP_METRICS_ENABLED == false

**Reply by k1eran** [reply]
@nirga Is there any programmatic way via `Traceloop.init()` to tell  openllmetry not to send to the OpenTelemetry Metrics, or it only possible via  the `TRACELOOP_METRICS_ENABLED` environment ? Thanks!

**Reply by nirga** [reply]
Currently only with the environment variable @k1eran. Can you open an issue for that? We'll take care of it

**Reply by k1eran** [reply]
@nirga Thanks for quick response. I created the feature request at https://github.com/traceloop/openllmetry/issues/1646
